gridData/OpenDX.py

```diff
@@ -166,6 +166,7 @@
 from six import next
 from six.moves import range
 import gzip
+import bz2
```

Comment:
If you allow bz2 then that also needs to be available when writing.

Reply:
Since we do not have code to read bz2 files right now, at least I can only find code to read .gz files, I would recommend we leave that out of this PR. @eloyfelix If you like you can later open a second PR to add the bz2.

Reply:
makes sense, I think that I'll remove it for now...